contracts/TokenSale.sol

```diff
@@ -329,6 +321,11 @@ contract TokenSale is FinalizableCrowdsale, Pausable {
         uint256 investedEthRefund = ethInvestments[msg.sender];
         uint256 investedStarRefund = starInvestments[msg.sender];
 
+        require(
```

Comment:
This is no needed as the funds are already set to 0 at the bottom. If you want to add just for extra security it is fine by me.

Reply:
not extra security, but to refund the gas